contrib/babelfishpg_common/src/collation.h

```diff
@@ -99,7 +99,7 @@ typedef struct collation_callbacks
 
 	bytea	   *(*tdscollationproperty_helper) (const char *collationaname, const char *property);
 
-	bool		(*is_server_collation_CI_AS) (void);
+	bool		(*is_database_or_server_collation_CI) (void);
```

Comment:
Can we give comments why we have this naming ? Personally I'm confused with this name.

Reply:
Added comment before function definition in .c files